src/util/email.js

```diff
@@ -302,34 +293,42 @@ module.exports = (formio) => {
       switch (emailType) {
         case 'default':
           if (_config && formio.config.email.type === 'sendgrid') {
-            transporter = nodemailer.createTransport(sgTransport({
-              apiKey: formio.config.email.password
-            }));
-          }
-          else if (_config && formio.config.email.type === 'mandrill') {
-            transporter = nodemailer.createTransport(mandrillTransport({
+            transporter = nodemailer.createTransport({
+              host: 'smtp.sendgrid.net',
+              port: 465,
+              secure: true,
               auth: {
-                apiKey: formio.config.email.apiKey,
-              },
-            }));
+                user: 'apikey',
+                pass: settings.email.sendgrid.auth.api_key
+              }
+            });
           }
           break;
         case 'sendgrid':
           if (_.has(settings, 'email.sendgrid')) {
             debug.email(settings.email.sendgrid);
-            transporter = nodemailer.createTransport(sgTransport({
-              apiKey: settings.email.sendgrid.auth.api_key
-            }));
-          }
-          break;
-        case 'mandrill':
-          if (_.has(settings, 'email.mandrill')) {
-            transporter = nodemailer.createTransport(mandrillTransport(settings.email.mandrill));
+            transporter = nodemailer.createTransport({
+              host: 'smtp.sendgrid.net',
+              port: 465,
```

Comment:
what made us choose implicit TLS rather than the recommend startTLS port 587? Will this mean that we can't use plain text at all via this port?

Reply:
This portion feels strange since we already give option for SMTP configuration and this basically abstracts that but does not leave them the ability to configure. I see what you mean though, I naively thought since we were abstracting away for this case and was succeeding with the implicit ssl/tls test, having it enabled would be beneficial.

I read a bit more and would agree with your assessment now and 587 would more likely be considered the default port to use.

[Mailgun - Port 587 is the default mail submission port](https://www.mailgun.com/blog/email/which-smtp-port-understanding-ports-25-465-587/#:~:text=Port%20587%3A%20The%20default%20port,-In%20December%20of&text=The%20RFC%20proposed%20a%20split,over%20message%20relay%20port%2025.)